{"links": {"self": {"href": "data/repositories/osrf/subt/issues/192/comments/54000113.json"}, "html": {"href": "#!/osrf/subt/issues/192#comment-54000113"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/192.json"}}, "type": "issue", "id": 192, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Image Solution runs with catkin setup but not using docker compose"}, "content": {"raw": "Hi @{5cf75629ff02b50ea27b2817} , \n\nThanks for taking the time. To answer your first question, yes I am running the catkin method and docker-compose on the same local machine. The error is that I have a check to see if there are cuda capable devices as such:\n\n`cudaError_t\u00a0status\u00a0=\u00a0cudaGetDevice(&n);`\n\n`assert(status ==\u00a0cudaSuccess);` \n\n\u200c\n\nAnd that gives me the error of assert\\(0\\), meaning there are no devices.  \n\nDocker -v  \n\nDocker version 19.03.2, build 6a30dfc\n\ndocker-compose -v\n\ndocker-compose version 1.23.2, build 1110ad01\n\nAnd I DO have the content on the daemon.json file. \n\nI believed I tested this capability in the past with you as well regarding other issue when you created the ./run\\_docker\\_compose.sh file but now seems to not work anymore. \n\n\u200c", "markup": "markdown", "html": "<p>Hi <span class=\"ap-mention\" data-atlassian-id=\"5cf75629ff02b50ea27b2817\">@Alfredo Bencomo</span> , </p>\n<p>Thanks for taking the time. To answer your first question, yes I am running the catkin method and docker-compose on the same local machine. The error is that I have a check to see if there are cuda capable devices as such:</p>\n<p><code>cudaError_t\u00a0status\u00a0=\u00a0cudaGetDevice(&amp;n);</code></p>\n<p><code>assert(status ==\u00a0cudaSuccess);</code> </p>\n<p>\u200c</p>\n<p>And that gives me the error of assert(0), meaning there are no devices.  </p>\n<p>Docker -v  </p>\n<p>Docker version 19.03.2, build <a href=\"#!/osrf/subt/commits/6a30dfc\" rel=\"nofollow\" class=\"ap-connect-link\">6a30dfc</a></p>\n<p>docker-compose -v</p>\n<p>docker-compose version 1.23.2, build <a href=\"#!/osrf/subt/commits/1110ad01\" rel=\"nofollow\" class=\"ap-connect-link\">1110ad01</a></p>\n<p>And I DO have the content on the daemon.json file. </p>\n<p>I believed I tested this capability in the past with you as well regarding other issue when you created the ./run_docker_compose.sh file but now seems to not work anymore. </p>\n<p>\u200c</p>", "type": "rendered"}, "created_on": "2019-09-20T12:01:01.658520+00:00", "user": {"display_name": "Hector Escobar", "uuid": "{1c140a49-f2c5-4727-abd9-eb427bfe4253}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1c140a49-f2c5-4727-abd9-eb427bfe4253%7D"}, "html": {"href": "https://bitbucket.org/%7B1c140a49-f2c5-4727-abd9-eb427bfe4253%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/8da90eb0e9b47c9adfc2893e5190483fd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsHE-1.png"}}, "nickname": "hector_escobar", "type": "user", "account_id": "5b5b1c941394ae6a72c021df"}, "updated_on": null, "type": "issue_comment", "id": 54000113}